doc/design/OrionMongoSink.md

```diff
@@ -90,9 +96,9 @@ A configuration example could be:
     cygnusagent.sinks.mongo-sink.type = com.telefonica.iot.cygnus.sinks.OrionMongoSink
     cygnusagent.sinks.mongo-sink.channel = mongo-channel
     cygnusagent.sinks.mongo-sink.mongo_hosts = 192.168.80.34:27017
-    cygnusagent.sinks.mongo-sink.mongo_username = admin
-    cygnusagent.sinks.mongo-sink.mongo_password = 1a2b3c4d
-    cygnusagent.sinks.mongo-sink.data_model = collection-per-service-path | collection-per-entity | collection-per-attribute
+    cygnusagent.sinks.mongo-sink.mongo_username = myuser
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5b6a45eb7c12d4db2e56ac0147eb6851b51c3d9